brd-ios/breadwallet/src/Redesign/Views/MainSwapView.swift

```diff
@@ -0,0 +1,201 @@
+// 
+//  MainSwapView.swift
+//  breadwallet
+//
+//  Created by Kenan Mamedoff on 05/07/2022.
+//  Copyright © 2022 Fabriik Exchange, LLC. All rights reserved.
+//
+//  See the LICENSE file at the project root for license information.
+//
+
+import UIKit
+import SnapKit
+
+struct MainSwapConfiguration: Configurable {
+    var shadow: ShadowConfiguration?
+    var background: BackgroundConfiguration?
+}
+
+struct MainSwapViewModel: ViewModel {
```

Comment:
way more properties are needed here.. like from currency, to currency,...

Reply:
![image](https://user-images.githubusercontent.com/99484553/177526415-d46bd339-b53e-445c-94b5-4955463f8943.png)

for now i have those. will extend when we wire the logic